### ISSUE TITLE
wsa_pacman 1.50 offline error

### ISSUE THREAD

**Issue by Noobability-noob** [alesimula/wsa_pacman#136]
even though i enabled developer settings, USB debuging, typed in the correct port number  and turned on WSA. It is still displaying the offline error. Can anyone please tell me how to fix this ??? (by the way, this happens on the portable version).

**Comment by Nzalo** [comment]
Hi, same problem here. As well running the portable version

**Comment by alesimula** [comment]
@maxgamer374 Did you get a pop-up window from WSA to allow ADB access to the computer? If not, try to open developer settings (NOT WSA Settings), scroll down and disable ADB debugging from there, then enable it again and retry (as explained in the FAQ)

**Comment by Noobability-noob** [comment]
@alesimula Yes, and i allowed ADB access i instaled 1 app and the next day it just... stoped working.

**Comment by alexivanov-ai** [comment]
I have the same problem with the installed version

**Comment by TitanComputer** [comment]
see my fix here https://github.com/alesimula/wsa_pacman/issues/95#issuecomment-2286991731
